src/controllers/conversation.controller.ts

```diff
@@ -0,0 +1,22 @@
+import {
+    JsonController,
+    Get,
+    Authorized,
+    CurrentUser,
+  } from 'routing-controllers';
+  import { Service } from 'typedi';
+import { ConversationsService } from '@services/conversation.service';
+import { Conversation } from '@entities/conversation.entity';
+  
+  @JsonController('/conversations')
+  @Service()
```

Comment:
Here you dont need the Service decorator. Im not expert but, Service decorator is useful to define a class like a dependency that it will be inject, in this case, the controller is not going to be used as an injectable dependency.

Reply:
Thank you for your feedback, to be honest I don't know much about this and I added the Service decorator because the original code had it in its controllers. I will learn more about the Service decorator.

Reply:
It seems that it's necessary to include the Service decorator to the controller:
"Don't forget to annotate your classes with the @Service decorator! Both the ones being injected and those which requests the dependencies should be annotated.". This is from the typedi documentation (https://docs.typestack.community/typedi/v/develop/02-basic-usage-guide)